modules/sbt-coursier-shared/src/main/scala/coursier/sbtcoursiershared/InputsTasks.scala

```diff
@@ -109,33 +109,33 @@ object InputsTasks {
       .toVector
       .flatMap(Inputs.ivyXmlMappings)
 
-    def dependency(conf: Configuration, attr: Attributes) = Dependency(
+    def dependency(conf: Configuration, pub: Publication) = Dependency(
       module,
       id.getRevision,
       conf,
       exclusions,
-      attr,
+      pub,
       optional = false,
       desc.isTransitive
     )
 
-    val attributes: Configuration => Attributes = {
+    val publications: Configuration => Publication = {
 
       val artifacts = desc.getAllDependencyArtifacts
 
       val m = artifacts.toVector.flatMap { art =>
-        val attr = Attributes(Type(art.getType), Classifier(""))
+        val pub = Publication("", Type(art.getType), Extension(""), Classifier(""))
```

Comment:
Didn't you intend to use `art.getName` and `art.getExt` here?

Reply:
If that's the case, do you have a case in mind where this is useful?

Reply:
I think `art.getName`, `art.getExt` makes more sense.